lib/lightning_web/live/job_live/job_builder.ex

```diff
@@ -226,7 +228,7 @@ defmodule LightningWeb.JobLive.JobBuilder do
             to: @return_to
           ) %>
           <Form.submit_button
-            disabled={!@changeset.valid?}
+            disabled={!@changeset.valid? or !@can_edit}
```

Comment:
Hmm, I find this double not with an or pretty confusing - it does appear to do the right thing..

Would it be clearer as `!(@can_edit && @changeset.valid?)`?
Like staggering the importance, "they can edit something" and then "is it valid"; and then inverting whatever the outcome of that is for `disabled`..